copilot_proxy/app.py

```diff
@@ -2,7 +2,7 @@
 
 import uvicorn
 from fastapi import FastAPI, Response
-from sse_starlette.sse import EventSourceResponse
+from fastapi.responses import StreamingResponse
```

Comment:
BTW, could you show us your self-assessment result with this PR?

Reply:
Do you mean #72 ?